schema/json/metaschema-datatypes.json

```
@@ -6,7 +6,7 @@
   "definitions" : {
     "Base64Datatype": {
       "type": "string",
-      "pattern": "^[0-9A-Fa-f]+$",
+      "pattern": "^[0-9A-Za-z+/]+={0,2}$",
```

B: Which flavors of Regex are we targeting (a readme or ADR documenting this would be helpful).

This regex pattern is invalid in PRCRE derived flavors.

---

This should fix the regex, which should still work well with ECMA.

```suggestion
      "pattern": "^[0-9A-Za-z+\/]+={0,2}$",
```

ECMA regex is required here, since it is JSON. In general we are targeting PCRE, Java, ECMA, XMLSchema flavors. I agree that an ADR on this would be useful. Creating an ADR is beyond the scope of this PR. Maybe an issue can be created for this?